### PR TITLE
Add tables

### PR DESCRIPTION
**Pull request by safar1212**
In this pull request, I implemented the following

Create a new app. Give it a meaningful name.
Initialize your project with Git.
Make sure that your project has Postgres database set up.
Build your project schema.
Create and run the necessary migration files.
Table and column names should match the ERD diagram.
note: photo for users table should be a link to an image
Foreign keys should be included.
All columns that are foreign keys should have a corresponding index.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,24 @@
-# blog-app
\ No newline at end of file
+# README
```
Comment (brytebee) [inline README.md]: - [ ] **Required:** Kindly update your README with the required columns in the [Microverse REAME template.](https://github.com/microverseinc/readme-template)

File: config/database.yml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  timeout: 5000
+
+development:
+  <<: *default
+  database: MyDatabase
+  encoding: unicode
+  host: localhost
+  pool: 5
+  username: postgres
+  password: admin
```
Comment (brytebee) [inline config/database.yml:19]: - [ ] **Optional:** In future projects, especially real apps, it is conventional to use `ENV` - environmental variables to hold your username and password. For development, the username and/or password for another developer varies from yours and using ENV ensures that any other developer can use your app without manually tempering your setup. [See this video](https://youtu.be/O-aDLsuNTRY) and do a little google search around this topic

File: config/database.yml [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+test:
+  <<: *default
+  database: db/test.sqlite3
+
+production:
+  <<: *default
+  database: MyDatabse
+  username: postgres
+  host: localhost
+  password: <%= ENV["MYDATABASE_PASSWORD"] %>
```
Comment (brytebee) [inline config/database.yml:33]: This is a use case for what I explained above.

File: db/migrate/20220914155226_create_user.rb [hunk 4]
```diff
@@ -0,0 +1,12 @@
+class CreateUser < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :Name
+      t.string :Photo
+      t.text :Bio
+      t.integer :PostsCounter
```
Comment (brytebee) [inline db/migrate/20220914155226_create_user.rb:7]: - [ ] **Optional:** Going forward, kindly note that migration files follow and `snake_case` naming convention and not `CamelCase` as you currently have. [See more here.](https://gist.github.com/iangreenleaf/b206d09c587e8fc6399e)

**Review by brytebee — CHANGES_REQUESTED** [review]
Hi, @safar1212👋,

##

### Highlights💡
- Good commit message.✔️ 
- Good PR Summary.✔️
- Great UI design.✔️
- App functions as it should.✔️

##

Good job so far coming so far with the progress you've made! 💪💪
 
However, there are some issues that you still need to work on to go to the next project but you are almost there!



# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Please, remember to tag me in your question so I can receive the notification.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brytebee in your question so I can receive the notification.
### _**Use @brytebee on slack for faster responses.**_


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by aimemalaika — APPROVED** [review]
Hi @safar1212 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 🎉

- No linter errors. ✔️
- GitHub flow is followed. ✔️
- Professional README file. ✔️
- PR has a good title and summary. ✔️

133591850-fdcb7dd3-dd7a-4274-81e7-0e59434148f7

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**